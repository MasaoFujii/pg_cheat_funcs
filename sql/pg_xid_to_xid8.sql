CREATE EXTENSION pg_cheat_funcs;

\pset null '(null)'

BEGIN;
SELECT 1 FROM pg_current_xact_id();
SELECT pg_current_xact_id() = pg_xid_to_xid8(backend_xid)
  FROM pg_stat_activity WHERE pid = pg_backend_pid();
COMMIT;

DROP EXTENSION pg_cheat_funcs;
