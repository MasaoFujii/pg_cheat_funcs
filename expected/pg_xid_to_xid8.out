CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
BEGIN;
SELECT 1 FROM pg_current_xact_id();
 ?column? 
----------
        1
(1 row)

SELECT pg_current_xact_id() = pg_xid_to_xid8(backend_xid)
  FROM pg_stat_activity WHERE pid = pg_backend_pid();
 ?column? 
----------
 t
(1 row)

COMMIT;
DROP EXTENSION pg_cheat_funcs;
